lib/WebDriverExpectedCondition.php

```diff
@@ -589,28 +589,31 @@ function (WebDriver $driver) use ($condition) {
      * An expectation with the logical AND condition of the given conditions.
      *
      * @param WebDriverExpectedCondition[] $conditions Array of the conditions to be satisfied.
-     * @return static Are all conditions satisfied or not.
+     * @return static Array of the getApply() values of all satisfied conditions or FALSE if some condition failed.
```

Comment:
```suggestion
     * @return static|false Array of the getApply() values of all satisfied conditions or FALSE if some condition failed.
```

Reply:
To be precise these methods always return `static` and the comments are related to `apply` return values